### PR TITLE
ss-326: Connect transactions in wallet and budget details pages to backend

### PR DESCRIPTION
**Pull request by Holovashcenko**
image

### CODE REVIEW

File: frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx [hunk 1]
```diff
@@ -172,6 +101,7 @@ const BudgetDetails = (): JSX.Element => {
     useEffect(() => {
         void dispatch(budgetsActions.loadAll());
         void dispatch(categoriesActions.loadCategories());
+        void dispatch(loadTransactions());
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx:104]: transactionActions.loadTransactions();

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx [hunk 2]
```diff
@@ -89,13 +92,19 @@ const WalletDetails: React.FC = () => {
     });
 
     useEffect(() => {
+        void dispatch(loadTransactions());
```
Comment (Evcat6) [inline frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx:95]: same

**Review by Evcat6 — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/common/components/transanction-table/components/transaction/transaction.tsx [hunk 3]
```diff
@@ -6,21 +8,21 @@ import {
...
-import OrangeMurseIcon from '~/assets/img/orange-murse-icon.svg';
 import { Input } from '~/bundles/common/components/components';
 import { getDaysLeft } from '~/bundles/common/components/transanction-table/helpers/index';
 import { InputType } from '~/bundles/common/enums/enums';
 
 import { type ITransaction } from '../../types';
 import styles from '../styles.module.scss';
 
-interface TransactionProperties {
+interface Properties {
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/common/components/transanction-table/components/transaction/transaction.tsx:18]: we are using type in our project

File: frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx (hunk 1 above)
Comment (Holovashcenko) [reply frontend/src/bundles/common/pages/budgets/budget-details/budget-details.tsx]: fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/41d8ea7e508c7c45c1cfa09406fa3012e4fce354](url)

File: frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx (hunk 2 above)
Comment (Holovashcenko) [reply frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx]: fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/41d8ea7e508c7c45c1cfa09406fa3012e4fce354](url)

File: frontend/src/bundles/common/components/transanction-table/components/transaction/transaction.tsx (hunk 3 above)
Comment (Holovashcenko) [reply frontend/src/bundles/common/components/transanction-table/components/transaction/transaction.tsx]: fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/21ca00bfc63bfbb4356b47d30687f448c8225403](url)

**Review by anton-otroshchenko — APPROVED** [review]

**Review by Evcat6 — APPROVED** [review]
good

**Pull request merged**